cli/cmd/download/plugin/cmd.go

```diff
@@ -49,27 +59,21 @@ This command fetches a specific plugin resource from the given OCM component ver
 The plugin binary can be identified by resource name and version, with optional extra identity parameters for platform-specific binaries.
 
 Resources can be accessed either locally or via a plugin that supports remote fetching, with optional credential resolution.`,
-		Example: ` # Download a plugin binary with resource name 'ocm-plugin' and version 'v1.0.0'
-  ocm download plugin ghcr.io/org/component:v1 --resource-name ocm-plugin --resource-version v1.0.0 --output ./plugins/ocm-plugin
-
-  # Download a platform-specific plugin binary with extra identity parameters
-  ocm download plugin ghcr.io/org/component:v1 --resource-name ocm-plugin --resource-version v1.0.0 --extra-identity os=linux,arch=amd64 --output ./plugins/ocm-plugin-linux-amd64
+		Example: ` # Download a plugin binary with resource name 'helminput' and version 'v0.0.0-main'
+  ocm download plugin ghcr.io/open-component-model/plugins//ocm.software/plugins/helminput:0.0.0-main
 
-  # Download plugin using only resource name (uses component version if resource version not specified)
-  ocm download plugin ghcr.io/org/component:v1 --resource-name ocm-plugin --output ./plugins/ocm-plugin`,
+  # Download a platform-specific plugin binary with extra identity parameters with specified output location.
+  ocm download plugin ghcr.io/open-component-model/plugins//ocm.software/plugins/helminput:0.0.0-main --extra-identity os=linux,arch=amd64 --output ./plugins/ocm-plugin-linux-amd64`,
 		RunE:              DownloadPlugin,
 		DisableAutoGenTag: true,
 	}
 
-	cmd.Flags().String(FlagResourceName, "", "name of the plugin resource to download (required)")
 	cmd.Flags().String(FlagResourceVersion, "", "version of the plugin resource to download (optional, defaults to component version)")
-	cmd.Flags().String(FlagOutput, ".", "output location to download the plugin binary to (required)")
+	cmd.Flags().String(FlagOutput, "", "output folder to download the plugin binary to (default $HOME/.config/ocm/plugins)")
```

Comment:
wouldn't it make sense to set the default to `pluginDirectoryDefault` that we can omit the empty check with fallback later?
```suggestion
	cmd.Flags().String(FlagOutput, pluginDirectoryDefault", "output folder to download the plugin binary to (default $HOME/.config/ocm/plugins)")
```